lib/auth/users/usersv1/service.go

```diff
@@ -554,7 +554,7 @@ func (s *Service) ListUsers(ctx context.Context, req *userspb.ListUsersRequest)
 			}); err != nil {
 				s.logger.WithError(err).Warn("Failed to emit local login failure event.")
 			}
-			return nil, trace.AccessDenied("this request can be only executed by an admin")
+			return nil, trace.AccessDenied("User secret information requires admin access and is not available from remote acces")
```

Comment:
```suggestion
			return nil, trace.AccessDenied("reading User secrets requires admin access and is not available from remote access")
```

Note the typo on "acces" too.